src/components/ProposalForm.tsx

```diff
@@ -112,54 +113,10 @@ const ProposalForm = forwardRef<ProposalFormMethods, ProposalFormProps>(
                   }
                 />
               ) : null}
-              <div className="sm:grid sm:grid-cols-4 sm:items-start sm:gap-4 sm:py-6">
-                <label
-                  htmlFor="title"
-                  className="block text-sm font-medium leading-6 text-gray-900 sm:pt-1.5"
-                >
-                  Title
-                </label>
-                <div className="mt-2 sm:col-span-3 sm:mt-0">
-                  <input
-                    type="text"
-                    name="title"
-                    id="title"
-                    placeholder="Title"
-                    className="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-teal-600 sm:max-w-sm sm:text-sm sm:leading-6"
-                  />
-                </div>
-              </div>
 
-              <div className="sm:grid sm:grid-cols-4 sm:items-start sm:gap-4 sm:py-6">
-                <label
-                  htmlFor="description"
-                  className="block text-sm font-medium leading-6 text-gray-900 sm:pt-1.5"
-                >
-                  Description
-                </label>
-                <div className="mt-2 sm:col-span-3 sm:mt-0">
-                  <textarea
-                    id="description"
-                    name="description"
-                    rows={3}
-                    className="block w-full max-w-2xl rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-teal-600 sm:text-sm sm:leading-6"
-                    defaultValue={""}
-                    placeholder="Description"
-                  />
-                  <p className="mt-3 text-sm leading-6 text-gray-600">
-                    Write a few sentences about the proposal and include any
-                    relevant links. Before proposing to Mainnet, please ensure
-                    you've started a discussion on the{" "}
-                    <a
-                      className="cursor-pointer hover:text-gray-900 underline"
-                      href={governanceForumLink}
-                    >
-                      Community Forum
-                    </a>
-                    .
-                  </p>
-                </div>
-              </div>
+              <TitleDescriptionInputs
```

Comment:
I keep scrolling down to **Sign & Submit** and hitting the button and getting no feedback... because I haven't selected  a chain nor connected a wallet. It would be nice if that top bar would stay put when I scroll. IOU an issue, I suppose.

Reply:
When I hit **Sign & Submit** (or something?) the console shows:

```
SES_UNHANDLED_REJECTION: TypeError: Cannot read properties of null (reading 'startsWith')
  at handleVaultSubmit (inter.tsx:104:28)
```

I suppose it's because I don't have an IBC denom?

Reply:
looks like `&&` should be `||`

```js
      if (!denom && !denom.startsWith("ibc/")) {
```

Reply:
``&&` -> `||` [70e0089](https://github.com/DCFoundation/cosmos-proposal-builder/pull/30/commits/70e00895615b660153e4a7382088e97887836b60)

> because I haven't selected a chain nor connected a wallet

added a toast warning when wallet or network are not connected: [531771a](https://github.com/DCFoundation/cosmos-proposal-builder/pull/30/commits/531771a8e6989338dd9ab9fdcecd9b60c9061b56)

Agree a sticky nav would be nice. There's also a ticket in the design backlog to have the connect wallet button also appear by "sign and submit":
<img width="627" alt="image" src="https://github.com/DCFoundation/cosmos-proposal-builder/assets/11021913/0cf9a46c-943c-41ff-8f55-a9266a53a3dd">

Reply:
re && -> || ... I'd like to see a test, if it's not too expensive to add.

> I haven't made strict habit of test-driven development, but ... I do make it a rule to write tests before fixing any bugs.
 -- [yours truly, 2019](https://www.madmode.com/2019/python-eng.html)

Reply:
tracking here: https://github.com/DCFoundation/cosmos-proposal-builder/issues/34